cryptpad-docker.md

```diff
@@ -0,0 +1,69 @@
+# Cryptpad Docker Image
+
+- Configuration via .env file
+- Ready for use with traffic
+- Using github master for now, release 0.3.0 too old
+- Creating customize folder
+- Adding config.js to customize folder
+- Persistance for datastore and customize folder
+
+## TODO
+
+```
+cryptpad_1  | Linking config.js
+cryptpad_1  | Using secure websockets: true
+cryptpad_1  | Using storage adapter: './storage/file'
+cryptpad_1  | sed: -e expression #1, char 27: unknown option to `s'
```

Comment:
Sed expects a backup suffix for the -i option. Write something like -i.bak and drop this .bak file afterwards